### DISCUSSION TITLE
Welcome to GitLearn Discussions!

### DISCUSSION THREAD

**Discussion by florian-corby** [florian-corby/GitLearn#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by frafraA** [comment]
Hi, my name is Amandine and I'm a beta testeur !!

**Reply by florian-corby** [reply]
A GREAT beta tester ;)

**Comment by florian-corby** [comment]
Je teste la réponse via ma boîte mail

**Comment by frafraA** [comment]
Je te réponds via mail

Télécharger Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by frafraA** [comment]
L'avalanche de mails !!

Télécharger Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________
